New package: Mice v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: Mice
- Repository: https://github.com/tom-metherell/Mice.jl
- Created by: @tom-metherell
- Version: v0.3.0
- Commit: c3ba70e3031cca1295688f8666b423d23d3c0793
- Reviewed by: @tom-metherell
- Reference: https://github.com/tom-metherell/Mice.jl/commit/c3ba70e3031cca1295688f8666b423d23d3c0793#commitcomment-134533373
- Description: a package for missing data handling via multiple imputation by chained equations in Julia. It is heavily based on the R package {mice} by Stef van Buuren, Karin Groothuis-Oudshoorn and collaborators.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
This is the first registered version of `Mice.jl`.

## Breaking changes
* Removed internal multithreading option (i.e. no more `threads` argument).

## Other major changes
* Improved overall performance significantly and
* Removed now unused internal functions.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68e4f4f14d75e1a5226963bd12cfd1f966f5f3ae13e6c810979bd29b95e9970f75b57434f720311af8192e5a7c941aaf8c9eb9887ef4dfe9a37c0cad0ab8d40542ad3305a69aeaeb5ae26f820e468013828047a3271c290e57690f21ca00b871cc9425397f8445085595856cc94d08a040926685e46d1fb998dda0f2f8225ba241b312fda75dc2a9ef2faad0ae819e873a7545c0af2fadbe4df7b674eccaf024d95fce594bda22820414c251ce7ac9dfa -->

tom-metherell commented:
Because this package is heavily based on an [R package of the same name](https://github.com/amices/mice), I believe that the name should be `Mice.jl`. So I am requesting a manual PR merge please :)

goerz commented:
The name seems fine to me. Ping the Slack channel in 3 days.

JuliaRegistrator committed:
New package: Mice v0.3.0

UUID: d4678d24-b338-4f96-a2c8-a66549d61c16
Repo: https://github.com/tom-metherell/Mice.jl.git
Tree: 0e6f31372c256207a3bfdea6e6584709875939af

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
